### ISSUE TITLE
javadoc deprecation warnings

### ISSUE THREAD

**Issue by gvanbrakel** [frankframework/frank-doc#110]
With each build of the Frank!Framework the following warnings are displayed when executing the FrankDoc:
```
Warning:  Javadoc Warnings
Warning:  javadoc: warning - The old Doclet and Taglet APIs in the packages
Warning:  com.sun.javadoc, com.sun.tools.doclets and their implementations
Warning:  are planned to be removed in a future JDK release. These
Warning:  components have been superseded by the new APIs in jdk.javadoc.doclet.
Warning:  Users are strongly recommended to migrate to the new APIs.
```
Please analyze and do the needful to solve this.

**Comment by mhdirkse** [comment]
When I Google on ..., I get page https://docs.oracle.com/javase/9/docs/api/jdk/javadoc/doclet/package-summary.html. The warnings are saying that from Java 9 onwards, we need to use a new Doclet API. I do not think that the new API exists for JDK 8.

I think we should implement the wrapper API for the new doclet API. Then we will have the old Java 8 implementation of the wrapper API next to the new Java 9+ implementation. When we move to Java 9+ with the Frank!Framework, we switch to using the new Doclet API.

This is quite a big task. When should we do it?

**Comment by jkosternl** [comment]
Is solved by #166
